tests/Unit/NumericalAlgorithms/RootFinding/Test_TOMS748.cpp

```diff
@@ -54,59 +55,40 @@ SPECTRE_TEST_CASE("Unit.Numerical.RootFinding.TOMS748.Bounds",
   CHECK(std::abs(root - sqrt(2.0)) < abs_tol);
   CHECK(std::abs(root - sqrt(2.0)) / sqrt(2.0) < rel_tol);
 
-  // Check that close but correct bounds work
-  {
-    const double tight_upper = sqrt(2.0) + abs_tol;
-    const auto tight_root =
-        RootFinder::toms748(f_lambda, lower, tight_upper, abs_tol, rel_tol);
-    CHECK(tight_root == approx(root));
-  }
-  {
-    const double tight_lower = sqrt(2.0) - abs_tol;
-    const auto tight_root =
-        RootFinder::toms748(f_lambda, tight_lower, upper, abs_tol, rel_tol);
-    CHECK(tight_root == approx(root));
-  }
+  // Check that the other tight-but-correct bracket works
+  CHECK(RootFinder::toms748(f_lambda, 0.0, sqrt(2.0) + abs_tol, abs_tol,
+                            rel_tol) == approx(root));
 
   // Check that exception is thrown for various bad bracket possibilities
-  int exception_count = 0;
   try {
-    const double bad_tight_upper = sqrt(2.0) - abs_tol;
-    RootFinder::toms748(f_lambda, lower, bad_tight_upper, abs_tol, rel_tol);
+    RootFinder::toms748(f_lambda, 0.0, sqrt(2.0) - abs_tol, abs_tol, rel_tol);
     INFO("Expected rootfinder to fail because upper bound is too tight");
     CHECK(false);
-  } catch (...) {
-    exception_count++;
+  } catch (std::domain_error&) {
   }
```

Comment:
And then:
```
} catch (...) {
  CHECK(false);
}
```
to make sure we don't propagate an incorrect exception. If you know what the exception message should be it would be good to check that too :)

Apply the same for your other checks of this type